### DISCUSSION TITLE
Is it possible to see custom domain in published Aspire Dashboard?

### DISCUSSION THREAD

**Discussion by efonsecab** [microsoft/aspire#4427] [Q&A] [answered]
So, I'm doing test for publishing my .NET Aspire solution into Azure.
Everything works just fine.

I did notice however that, if I configure Custom Domains in the container app from within the portal, the Aspire Dashboard never shows the custom domain, is it possible somehow to make the dashboard recognize the custom domain?

 
![NET Aspire Published to Azure](https://github.com/dotnet/aspire/assets/3481899/e870ecc9-7130-402e-8bc9-fb4813f524d5)

**Comment by davidfowl** [comment, marked as answer]
No there’s no way to make them show up right now, but we’ll treat this like a feature request for the dashboard in ACA

**Reply by efonsecab** [reply]
Got it, thanks. Not a big deal, it would just be ideal.

**Comment by hectorcubillos** [comment]
Hey @davidfowl !
Any updates on this ?
